Readme.md

```diff
@@ -16,8 +16,8 @@
 ## #BLM and 5.x breaking changes
 
 The maintainers of this project no longer feel comfortable with the following terms:
-* whitelist
-* blacklist
+* allowlist
+* denylist
```

Comment:
In this instance it is referencing the previous terms so this should not be replaced